Portfolio/index.html

```diff
@@ -3,83 +3,79 @@
   <head>
     <title>Portfolio</title>
     <meta charset="UTF-8" />

```

Comment:
Consider adding crossorigin attribute to improve security when loading fonts from external sources